draft-barnes-acme.md

```diff
@@ -323,6 +323,12 @@ recoveryToken (optional, string):
 agreement (optional, string):
 : A URI referring to a subscriber agreement or terms of service provided by the server (see below).  Including this field indicates the client's agreement with these terms.
 
+authorizations (optional, string):
+: A URI from which a list of authorizations granted to this account can be fetched via a GET request.  The result of the GET request MUST be a JSON array of strings, where each string is the URI of an authorization belonging to this registration.
```

Comment:
Also, I'd suggest that the target URL should be a JSON object containing the single field 'authorizations', which maps to a list of strings. That lets us maintain the property that all ACME objects are JSON objects, not other types.